devicetypes/gs/gatorsystem-homewatcher.src/gatorsystem-homewatcher.groovy

```diff
@@ -0,0 +1,109 @@
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+metadata {
+	definition (name: "GatorSystem HomeWatcher", namespace: "GS", author: "GS_coder000") {
+	capability "Motion Sensor"
+	capability "Battery"
+	capability "Contact Sensor"
+	capability "Presence Sensor"   
+	// Raw Description
```

Comment:
Please add Raw Description content. As i mentioned before: 

It can be found at graph.api.smartthings.com -> My Devices -> Raw Description, so it would look like:
// Raw Description **insert content here**

Please let me know If You have any troubles finding it.